### PR TITLE
Add underscores to unused parameters

### PR DESCRIPTION
**Pull request by jelly**
oxlint complains about this by default, its `no-unused-vars` is stricter then eslint :)

### CODE REVIEW

File: src/components/vm/nics/vmNicsCard.jsx [hunk 1]
```diff
@@ -279,7 +279,7 @@ export class VmNetworkTab extends React.Component {
         this.getIpAddr();
     }
 
-    componentDidUpdate(prevProps, prevState) {
+    componentDidUpdate(prevProps, _prevState) {
```
Comment (martinpitt) [inline src/components/vm/nics/vmNicsCard.jsx:282]: It'd be cleaner to just entirely drop trailing unused arguments.

File: src/components/vm/vmCloneDialog.jsx [hunk 2]
```diff
@@ -60,7 +60,7 @@ export const CloneDialog = ({ name, connectionName }) => {
             options.superuser = "try";
         return cockpit.spawn(["virt-clone", "--connect", "qemu:///" + connectionName, "--original", name, "--name", newVmName, "--auto-clone"], options)
                 .stream(setVirtCloneOutput)
-                .then(Dialogs.close, exc => {
+                .then(Dialogs.close, _exc => {
```
Comment (martinpitt) [inline src/components/vm/vmCloneDialog.jsx:63]: and `()` here.

File: src/helpers.js [hunk 3]
```diff
@@ -296,7 +296,7 @@ export function rephraseUI(key, original) {
  * @param mimeType
  * @returns {*}
  */
-export function fileDownload({ data, fileName = 'myFile.dat', mimeType = 'application/octet-stream' }) {
+export function fileDownload({ data, _fileName = 'myFile.dat', mimeType = 'application/octet-stream' }) {
```
Comment (martinpitt) [inline src/helpers.js:299]: Ah, that's the commented "can't get CSP working" comment. Feels weird to have an unused argument with a default value, but maybe one day we'll figure this out :sweat_smile:

File: src/libvirtApi/common.js [hunk 4]
```diff
@@ -374,7 +374,7 @@ function startEventMonitorNetworks(connectionName) {
     /* Subscribe to signals on Network Interface */
     dbusClient(connectionName).subscribe(
         { interface: "org.libvirt.Network" },
-        (path, iface, signal, args) => {
+        (path, _iface, signal, _args) => {
```
Comment (martinpitt) [inline src/libvirtApi/common.js:377]: args could also just be dropped.

File: src/libvirtApi/common.js [hunk 5]
```diff
@@ -424,7 +424,7 @@ function startEventMonitorStoragePools(connectionName) {
     /* Subscribe to signals on StoragePool Interface */
     dbusClient(connectionName).subscribe(
         { interface: "org.libvirt.StoragePool" },
-        (path, iface, signal, args) => {
+        (path, _iface, signal, _args) => {
```
Comment (martinpitt) [inline src/libvirtApi/common.js:427]: Dito

**Review by martinpitt — APPROVED** [review]
Thanks! Depending on how much effort you want to throw at this, land as-is or one more round.

File: src/components/vm/snapshots/vmSnapshotsCreateModal.jsx [hunk 6]
```diff
@@ -134,7 +134,7 @@ function get_available_space(path, superuser, callback) {
                 const info = JSON.parse(output);
                 callback(info.free * info.unit);
             })
-            .catch(exc => {
+            .catch(() => {
```
Comment (cockpituous) [inline src/components/vm/snapshots/vmSnapshotsCreateModal.jsx:137]: This added line is not executed by any test.

File: src/components/vm/vmCloneDialog.jsx (hunk 2 above)
Comment (cockpituous) [inline src/components/vm/vmCloneDialog.jsx:63]: This added line is not executed by any test.

File: src/libvirtApi/common.js (hunk 4 above)
Comment (cockpituous) [inline src/libvirtApi/common.js:377]: This added line is not executed by any test.

File: src/helpers.js (hunk 3 above)
Comment (jelly) [reply src/helpers.js:299]: Kept this in because of indeed it is used in uncommented code :(

**Review by martinpitt — APPROVED** [review]
`_lgtm: true`

**Pull request merged**